helpers/stable_swap_math/mod.rs

```diff
@@ -826,54 +798,9 @@ mod tests {
 
     #[test]
     fn deposit_liquidity_by_share_and_by_amounts_equality_1() {
-        let amp_coef: u128 = 85;
-        let fees = Fees::new(2137, 0); // 10% fee
-        let reserves: Vec<u128> = Vec::from([500_000_000_000, 500_000_000_000]);
-        let token_supply = compute_d(&reserves, amp_coef).unwrap().as_u128();
-        let share = token_supply / 20; // 5%
-        let deposit_amounts = compute_amounts_given_lp(share, &reserves, token_supply)
-            .unwrap_or_else(|_| panic!("Should mint liquidity"));
-        let (share_by_deposit, fee_part) = compute_lp_amount_for_deposit(
-            &deposit_amounts,
-            &reserves,
-            token_supply,
-            Some(&fees),
-            amp_coef,
-        )
-        .unwrap_or_else(|_| panic!("Should mint liquidity"));
-        assert_eq!(fee_part, 0, "Fee should be 0");
-        assert_eq!(share, share_by_deposit, "Deposit amounts differ.");
-    }
-
-    #[test]
-    fn withdraw_liquidity_by_share_and_by_amounts_equality_2() {
-        let amp_coef: u128 = 85;
-        let fees = Fees::new(2137, 0); // 10% fee
-        let reserves: Vec<u128> = Vec::from([12341234123412341234, 5343245543253432435]);
-        let token_supply = compute_d(&reserves, amp_coef).unwrap().as_u128();
-        let share = token_supply / 20; // 5%
-        let withdraw_amounts_by_share = compute_amounts_given_lp(share, &reserves, token_supply)
-            .unwrap_or_else(|_| panic!("Should work"));
-        let (share_by_withdraw_amounts, fee_part) = compute_lp_amount_for_withdraw(
-            &withdraw_amounts_by_share,
-            &reserves,
-            token_supply,
-            Some(&fees),
-            amp_coef,
-        )
-        .unwrap_or_else(|_| panic!("Should work"));
-        assert_eq!(fee_part, 0, "Fee should be 0");
-        assert_eq!(
-            share_by_withdraw_amounts, share,
-            "Share amounts should match"
-        );
-    }
-
-    #[test]
-    fn deposit_liquidity_by_share_and_by_amounts_equality_2() {
```

Comment:
Why are you removing these tests?

Reply:
These tests just show that depending on the method used to calculate liquidity/swap the rounding error influences the result (they were meant to fail).

Reply:
Failing tests are also good, but if you think they didn't contribute to confidence in the code it's OK to remove.